### DISCUSSION TITLE
Cross compare model skills

### DISCUSSION THREAD

**Discussion by Hendrik1987** [DHI/modelskill#20] [Ideas]
Capacity to cross-compare QI parameters of different models / model calibration cases. I.e., adding a dimension to the skill report.
There are many ways to visualize this, this could be one example:
Location vs. bias of Hm0 for various calibration runs of a SW model:
![QIcompare](https://user-images.githubusercontent.com/29595985/108679583-097a1900-74ed-11eb-816c-186c33e21dac.png)

**Comment by Hendrik1987** [comment]
@jsmariegaard: After your demonstration yesterday, I now see that 'calibration' is already included as a use-case. So this idea is redundant, I leave it here for now for the example plot.

**Reply by jsmariegaard** [reply]
I think we should add some examples like this to a notebook. The data is all there by the skill() - but we should still show some good ways to visualize. Would it even make sense to do a Skill class similar to #41 for convenient plotting etc?

**Reply by jsmariegaard** [reply]
@Hendrik1987 this should now be "solved" with the plot_line() method in the AggregatedSkill class as of #47

**Reply by Hendrik1987** [reply]
@jsmariegaard: Great new features, yes, solved! 👍
